### DISCUSSION TITLE
Add modifier to enable scrollbar on carousel

### DISCUSSION THREAD

**Discussion by silveltman** [saadeghi/daisyui#1780] [Ideas / feature requests]
The scrollbar on a carousel is hidden by DaisyUI. Sometimes it's useful to show the scrollbar, therefore a modifier to do this would be great!

**Comment by alexclinky** [comment]
How do you show it? I can't figure out how to enable horizontal scroll bar?

**Reply by silveltman** [reply]
```
carousel::-webkit-scrollbar {
display: block !important;
}
```
or `[&::-webkit-scrollbar]:!block` for inline tailwind

You can have a look here: https://github.com/adoxography/tailwind-scrollbar/issues/70

**Comment by saadeghi** [comment]
@alexclinky Answered here: https://github.com/saadeghi/daisyui/discussions/1829

The problem is I couldn't find a way to undo the style for `::-webkit-scrollbar` When I use `display:none` to hide it, any other style won't turn it back to the original style. `display:block` makes it visible but without the original style so then we have to add custom size for `::-webkit-scrollbar` and custom colors for `::-webkit-scrollbar-thumb` to make it visible but it's basically a customized scrollbar (only for webkit) while the original scrollbar mimics the OS/browser style (it's different on MacOS/Windows/Linux)
